content/en/docs/setup/production-environment/container-runtimes.md

```diff
@@ -45,6 +45,37 @@ check the documentation for that version.
 
 
 <!-- body -->
+## {{% heading "prerequisites" %}}
+
+These instructions apply to Linux nodes only.
```

Comment:
re: the thread here:
https://github.com/kubernetes/website/pull/32946/files#r851613554

looks like CRIO also recommends overlay as the SD:
https://access.redhat.com/documentation/en-us/openshift_container_platform/3.11/html/cri-o_runtime/use-crio-engine
netfilter is also seen in external guides:
https://wiki.archlinux.org/title/CRI-O

with that i think the settings apply to * CRs that we support on this page.

suggesting the following, but language can be adjusted:

```suggestion
These instructions apply common container runtime settings for Kubernetes nodes on Linux.
If you are certain that your setup does not require a certain setting you can skip it.
Consult the individual container runtime documentation for further information.
```